modules/installation-configuration-parameters.adoc

```diff
@@ -407,6 +407,20 @@ endif::ibm-power-vs[]
 Set the `networking.machineNetwork` to match the CIDR that the preferred NIC resides in.
 ====
 
+|networking:
+  ovnKubernetesConfig:
+    ipv4:
+      internalJoinSubnet:
+|When you configure `networkType` to `OVNKubernetes`, `OVNKubernetesConfig` provides the configuration for ovn-kubernetes as the default pod network.
+
+You can configure the IP settings for IPv4 connections. If you do not configure IPv4, the default configuration is used. For more information about the default configuration values, see the individual IPv4 fields.
```

Comment:
I think this line is not necessary, since the `ipv4:` parameter is not really a parameter itself, but a category of parameters.